airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -1120,14 +1214,19 @@ protected void runSyncAndVerifyStateOutput(final JsonNode config,
                                              final ConfiguredAirbyteCatalog catalog,
                                              final boolean runNormalization)
       throws Exception {
-    final List<AirbyteMessage> destinationOutput = runSync(config, messages, catalog, runNormalization);
-    final AirbyteMessage expectedStateMessage = MoreLists.reversed(messages)
+    final List<AirbyteMessage> destinationOutput = runSync(config, messages, catalog,
+        runNormalization);
+
+    Collections.reverse(messages);
```

Comment:
`Collections.reverse` changes the order of `messages`.  `MoreLists.reversed` generates a new List with reversed messages, and the order of origin `messages` don't change.   

this code change may cause some integration/acceptance test fail